server/index.js

```diff
@@ -59,6 +59,26 @@ app.post('/api/hosts', uploadsMiddleware, (req, res, next) => {
     .catch(err => next(err));
 });
 
+app.get('/api/pools/:location', (req, res, next) => {
+  const location = req.params.location;
+  const sql = `
+    select "location",
+           "price",
+           "image",
+           "poolId"
+      from "pools"
+    where "location" = $1`;
+  const params = [location];
+  db.query(sql, params)
+    .then(result => {
+      if (!result.rows[0]) {
+        throw new ClientError(400, `No pools found in ${location}`);
+      }
```

Comment:
An empty list is fine. The request was successful and (maybe) later, there could be results for that location. 4xx errors mean that the client will _never_ get what they want with that request. Just show a "no results found" screen if the list is empty.